src/Passes/Source/Generators/IProfileGenerator.hpp

```diff
@@ -0,0 +1,133 @@
+#pragma once
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "Commandline/ConfigurationManager.hpp"
+#include "Profile/Profile.hpp"
+#include "Types/Types.hpp"
+
+#include "Llvm/Llvm.hpp"
+
+namespace microsoft
+{
+namespace quantum
+{
+
+    class IProfileGenerator
```

Comment:
Why does this class name (`IProfileGenerator`) start with `I`? 
The class does not have any pure virtual function (not even just virtual function).

(If you rename the class then don't forget to rename the files too)

Reply:
This is a leftover from something that started as an interface. However, you are right - it is not. I've removed it.